Makefile

```diff
@@ -48,9 +48,14 @@ TOOLS_DIR := internal/tools
 update-components:
 	grep -o github.com/open-telemetry/opentelemetry-collector-contrib/[[:lower:]]*/[[:lower:]]* go.mod | xargs -I '{}' go get {}
 	go mod tidy
-	cd $(TOOLS_DIR) && \
-		go get -u github.com/open-telemetry/opentelemetry-collector-contrib/cmd/mdatagen && \
-		go mod tidy
+	cd $(TOOLS_DIR) && go get -u github.com/open-telemetry/opentelemetry-collector-contrib/cmd/mdatagen
+	cd $(TOOLS_DIR) && go mod tidy
+
+update-opentelemetry: 
+	GO_BUILD_TAGS=gpu
+	$(MAKE) update-components 
+	$(MAKE) install-tools 
+	$(MAKE) generate
```

Comment:
I don't know that setting a variable in a separate command actually affects the subsequent commands — did you want to set this for one of the sub-commands instead?
```suggestion
	$(MAKE) update-components 
	$(MAKE) install-tools 
	$(MAKE) GO_BUILD_TAGS=gpu generate
```

Reply:
You're right, fixed it